CHANGELOG.md

```diff
@@ -19,6 +19,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Changed
 
 - Extra padding is not evenly spread around the terminal by default anymore
+- Uncaught panics in Windows now expose themselves by way of a classical Message Box in addition to dumping report to STDERR.
```

Comment:
Change log entries should not contain a dot at the end.

Reply:
Sorry about that.  I had thought I saved the buffer before committing.